desktop/linux/release-notes/index.md

```diff
@@ -10,6 +10,23 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
+## Docker Desktop 4.8.1
+2022-05-09
+
+> Download Docker Desktop
+>
+> [DEB](https://desktop-stage.docker.com/linux/main/amd64/78933/docker-desktop-4.8.0-amd64.deb) |
```

Comment:
the three download links need some care to update to the new build number and version number.

Reply:
Thanks! It's been updated.